sklearn/metrics/tests/test_common.py

```diff
@@ -368,7 +368,6 @@
 
 # No Sample weight support
 METRICS_WITHOUT_SAMPLE_WEIGHT = [
-    "cohen_kappa_score",
```

Comment:
As far as I understand, no. It's already in CLASSIFICATION_METRICS, which is in ALL_METRICS, which means it will now be tested by test_sample_weight_invariance().